text/0000-export-to-oci.md

```diff
@@ -0,0 +1,192 @@
+- A [Lifecycle](https://buildpacks.io/docs/concepts/components/lifecycle/) orchestrates Buildpacks execution, then assembles the resulting artifacts into a final app image.
+- A Daemon is a service, popularized by Docker, for downloading container images, and executing and managing containers from those images.
+- A Registry is a long-running service used for storing and retrieving container images.
+- A digest reference refers to a [content addressable](https://en.wikipedia.org/wiki/Content-addressable_storage) identifier of form <registry>/<repo>@<digest> which locates an image manifest in an [OCI Distribution Specification](https://github.com/opencontainers/distribution-spec/blob/master/spec.md) compliant registry.
+- A Image Manifest provides a configuration and set of layers for a single container image for a specific architecture and operating system.
+- An [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) is the directory structure for OCI content-addressable blobs and location-addressable references.
+
+# Motivation
+[motivation]: #motivation
+
+Implementing this new feature will help us to solve the problem of loosing information when the image is saved into the Daemon keeping the image on disk along with the metadata it can be used as input for other tools to offer more capabilities to the end users.
+
+This feature will help to unblock uses cases like
+- OCI annotations. See [RFC](https://github.com/buildpacks/rfcs/pull/196)
+- Cosign integration. See [RFC](https://github.com/buildpacks/rfcs/pull/195)
+
+# What it is
+[what-it-is]: #what-it-is
+
+Currently the *Exporter*  writes either in an OCI image registry or a docker daemon, the idea is to add the capability to write into disk in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format only when the `-daemon` flag is used as argument AND the feature is enable using a new flag `-layout` or the default environment variable `CNB_LAYOUT_DIR`.
+
+## Examples
+
+### Exporting using the environment variable
+
+```=shell
+> export CNB_LAYOUT_DIR=oci
+>  /cnb/lifecycle/exporter -daemon my-app-image
+> tree /oci
+.
+└── oci/
+    └── my-app-image/
+        ├── blobs/
+        │   └── sha256/
+        │       └── 01..
+        ├── index.json
+        └── oci-layout
+
+```
+
+### Exporting using the command line flag
+
+```=shell
+>  /cnb/lifecycle/exporter -daemon -layout oci my-app-image
+> tree /oci
+.
+└── oci/
+    └── my-app-image/
+        ├── blobs/
+        │   └── sha256/
+        │       └── 01..
+        ├── index.json
+        └── oci-layout
+
+```
+As we can see there is a new folder called `oci` and inside that folder we can find our application image in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format
+
+Attempts to use this feature when the `-daemon` flag is not used could be ignore or show some warnings messages.
```

Comment:
I don't think we need this if we allow a platform to use this `-layout` with any other configuration.